roles/freeipa_client/meta/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+# Copyright 2023 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+galaxy_info:
+  role_name: freeipa_client
+  namespace: cloudera
+  author: Webster Mudge (wmudge@cloudera.com)
```

Comment:
Update `authors`

Reply:
the author really is JIm, I will put him